### DISCUSSION TITLE
Saving to usb drive

### DISCUSSION THREAD

**Discussion by blogdemoi** [ArchiveBox/ArchiveBox#1141] [Q&A] [answered]
Hi,

How and where/what should I change to save to an external drive (usb attached)?

Thank you.

**Comment by pirate** [comment, marked as answer]
In the `docker-compose.yml` file, change the `volumes:` section:

```yaml

...
volumes:
  - /USB/Volume/path/here/archivebox:/data
...
```

`/Volume/path/here/archivebox` should be the path to your USB drive folder for archivebox. e.g. on macOS `/Volumes/USB NAME/archivebox`, or linux `/mnt/...` etc.

**Reply by blogdemoi** [reply]
Thank you, worked perfectly.
